src/popup/locales/en.json

```diff
@@ -1196,6 +1197,7 @@
     "notSameAs": "Sender and receiver address are the same. You will send {0} to your own account.",
     "tokenToAnAddress": "Unfortunately you can only send AEX-9 Tokens to an address",
     "addressBtc": "Invalid BTC address",
+    "addressEth": "Invalid ETH address",
```

Comment:
I strongly suggest to not create any protocol related translations.
I know that there is already `addressBtc` here, but I think we need to refactor this and pass the coin as property.